cmd/unregister.go

```diff
@@ -127,19 +100,59 @@ var unregisterCmd = &cobra.Command{
 						clusterRoleBinding.Subjects = append(clusterRoleBinding.Subjects[:k], clusterRoleBinding.Subjects[k+1:]...)
 						_, err = clientset.RbacV1().ClusterRoleBindings().Update(clusterRoleBinding)
 						checkError(err)
-						fmt.Printf("User %s unregistered on %s \n", email, cluster.Name)
+						fmt.Printf("User %s unregistered \n", email)
 						break
 					}
 				}
+			} else {
+				currentKubeConfig := getGardenKubeConfig()
+				var gardenConfig GardenConfig
+				GetGardenConfig(pathGardenConfig, &gardenConfig)
+				for _, cluster := range gardenConfig.GardenClusters {
+					gardenKubeConfig := cluster.KubeConfig
+					if strings.Contains(gardenKubeConfig, "~") {
+						gardenKubeConfig = filepath.Clean(filepath.Join(HomeDir(), strings.Replace(gardenKubeConfig, "~", "", 1)))
+					}
+					config, err := clientcmd.BuildConfigFromFlags("", gardenKubeConfig)
+					checkError(err)
+					clientset, err := k8s.NewForConfig(config)
+					checkError(err)
+					clusterRoleBinding, err := clientset.RbacV1().ClusterRoleBindings().Get("garden.sapcloud.io:system:administrators", metav1.GetOptions{})
```

Comment:
Could you please replace the role "garden.sapcloud.io:system:administrators" by a constant and replace it for all occurrences.